Automattic/wp-calypso
# Support: show ticket contact form when olark fails to load

This PR checks if olark failed to load in a timely manner, and falls back to the ticket contact form. This may happen if a user is using adblock plugins. This fixes #5338

Before:
<img width="767" alt="before" src="https://cloud.githubusercontent.com/assets/1270189/15201075/e7da18ca-17a1-11e6-9880-75f23c8b45ee.png">

After:
<img width="810" alt="failedtoload" src="https://cloud.githubusercontent.com/assets/1270189/15231793/1463a28e-1852-11e6-94b8-7d9eadb7eea0.png">

## Implementation Notes:
- I've added a window check around olark-api for easier testing
- Enough redux state was added to be able to check if olark did not load in a timely manner. We should have follow up PRs to port the rest of olark state from flux to redux.
- State was added in the UI tree (which is never persisted). Any olark information is unlikely to be useful past the current page load. If this changes, we'll want to move this out of that special subtree.

## Testing Instructions
- Use a browser with an adblock plugin like uBlock enabled
- Navigate to calypso.localhost:3000/help/contact
- Wait a while, the ticket contact form should render with a warning
- Disable the adblock plugin
- Refresh the page
- The contact form should appear

cc @DanReyLop @omarjackman @rralian @jordwest
